variables.tf

```diff
@@ -35,3 +35,30 @@ variable "region" {
     error_message = "Invalid input, options: \"us-south\", \"ca-tor\", \"eu-es\", \"eu-de\"."
   }
 }
+
+variable "cos_instance_guid" {
+  type        = string
+  description = "GUID of the Cloud Object Storage to store SCC data"
+}
+
+variable "cos_instance_id" {
```

Comment:
this should actually be `cos_instance_crn` as this is what SCC requires:
```
object_storage {
    instance_crn = "crn..."
    bucket       = "my-bucket"
  }
```

It works with ID because ID for COs is actually the CRN value, but to avoid confusion, lets rename the variable.